fbw-a32nx/src/systems/fmgc/src/index.ts

```diff
@@ -64,4 +66,6 @@ export {
   DataManager,
   EventBus,
   a320EfisRangeSettings,
+  A320AircraftConfig,
```

Comment:
Do these actually need to be exported? Does vanilla JS use it?

Reply:
not to my knowledge, removed the exports